lib/components/Notification/item/index.tsx

```diff
@@ -68,6 +68,7 @@ const NotificationItem = ({
                 "notification__item-wrapper",
                 isDeleted && "deleted",
             )}
+            onBlur={() => setShouldShowButtons(false)}
```

Comment:
moved `onBlur` from anchor to `div` since `onBlur` doesn't work on anchors on iOS